.github/workflows/python-publish.yml

```diff
@@ -34,5 +34,5 @@ jobs:
         TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
         TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
       run: |
-        python setup.py sdist bdist_wheel
+        python -m build  # builds both sdist and wheel
```

Comment:
Fantastic! This is way more elegant and easier to see what this line is actually doing. I want more simplicity like this. Keep up the good work!